### PR TITLE
Add CallABike

### PR DESCRIPTION
**Pull request by martgnz**
Closes https://github.com/eskerda/pybikes/issues/238.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ name: pybikes
 
 env:
   PYBIKES_CYCLOCITY: ${{ secrets.PYBIKES_CYCLOCITY }}
-  PYBIKES_ECOBICI_BA_CLIENT_ID: ${{ secrets.PYBIKES_ECOBICI_BA_CLIENT_ID }}
-  PYBIKES_ECOBICI_BA_CLIENT_SECRET: ${{ secrets.PYBIKES_ECOBICI_BA_CLIENT_SECRET }}
```
Comment (janpio) [inline .github/workflows/test.yml:6]: Expected unrelated change or mistake?

Comment (eskerda) [reply .github/workflows/test.yml:6]: Expected unrelated change 👌  this class was replaced by an unauthenticated one at https://github.com/eskerda/pybikes/pull/617
